applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -45,8 +44,6 @@
 TARGET_RELEASE_FOLDER = "build_release"
 TARGET_DEBUG_FOLDER = "build_debug"
 
-PRISTINE_FLAG = " --pristine"
```

Comment:
Why is this removed?

Reply:
> Why is this removed?

Because we don't use this DEFINE in buildprog.py.
When --pristine is assigned to buildprog.py, we use two methods to make a clean build.

1. remove build tree by ``shutil.rmtree(dest_folder)``.
2. ``west build -p`` this is the same thing ``west build --pristine``